doc/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting
+
+This documentation will help you to troubleshoot docker related issues while running your ccl unit test with "container" environment.
+
+- [Troubleshooting](#troubleshooting)
+    - [Docker Container](#docker-container)
+    - [Timeout errors](#timeout-errors)
+    - [Failed to start containers: ccloracle](#failed-to-start-containers-ccloracle)
+    
+### Docker Container
+If you are getting the following error while executing the ccl unit test in Gaia, it means that you are using a wrong container in docker.
+```sh
+"error: 'Error: Command failed: docker pull docker-snapshot.cernerrepos.net/mpages/ccltesting \nimage operating system \"linux\" cannot 
+be used on this platform\n' stdout = 'Using default tag: latest\nlatest: Pulling from mpages/ccltesting\n' stderr = 'image operating 
+system \"linux\" cannot be used on this platform\n'"
+```
+To resolve this issue, you need to change the container from Windows to Linux.
+1. Go to the docker settings and then click on the "Switch to Linux containers... " option.
+2. The docker will switch the container and then it will automatically restart itself.
+3. Once docker starts running, run your unit test again.
+
+### Timeout Errors
+If you are getting the following errors while executing the ccl unit test in Gaia, then it means that you might not be accessing the application within Cerner's Network.
+```sh
+"error: 'Error: Command failed: docker pull docker-snapshot.cernerrepos.net/mpages/ccltesting \nError response from daemon: Get 
+https://docker-snapshot.cernerrepos.net/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while 
+awaiting headers)\n' stdout = 'Using default tag: latest\n' stderr = 'Error response from daemon: Get https://docker-
+snapshot.cernerrepos.net/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting 
+headers)\n'"
+```
+To resolve this error, try fetching the docker images through Cerner network as they are available only for the internal network.
+
+### Failed to start containers: ccloracle
+
+If you encounter errors such as mentioned below, it means you have duplicate networks on which docker is running.
```

Comment:
Are these really "CCL Unit issues"?  Or are they simply "CCL issues" or even just "gaia issues" that would occur no matter what one is trying to do in CCL?   If they are not really CCL Unit specific issues, then perhaps this is not the best place to put these troubleshooting steps.

The following example is extreme but it illustrates the point of the question.  If printer XYZ is not configured correctly, then one will struggle when trying to execute `cclut "XYZ", "ccluserdir", "some_test_suite" go`. Should this repository provide troubleshooting steps explaining how to set up printers in case someone runs into that problem? Note that one would also struggle when trying to execute `select into "XYZ" cv.display from code_value cv where cv.code_set = 25 go`.  

Perhaps the instructions explaining how to set up and use gaia could/should provide a set of "test your configuration" steps providing various tests to check if everything is set up correctly along with corrective steps for when the tests fail.  One such test might "launch CCL and execute X within CCL".

Reply:
Yes this is not technically a CCL Unit test issue, but when someone uses CCL unit testing plugin and use docker to run the test, they might face this issue.
This is like if someone is trying to run a maven project without Java installed, the troubleshooting guide will have information to install Java before Maven to successfully run the project. I am sure troubleshooting guide cannot have all possible errors that one might face, but it is a good step to point out the common errors and provide solution for the same. I think this will be helpful for people who is using this plugin for the first time.

Reply:
I am not familiar with the referenced maven troubleshooting guide or the portion of it that points out the errors that will be seen if java is not installed and recommending to install java to resolve the matter. Can you provide a link to what you are talking about?